tools/eslint/node_modules/graceful-fs/polyfills.js

```diff
@@ -1,5 +1,4 @@
 var fs = require('./fs.js')
-var constants = require('constants')
```

Comment:
Whoops! Picked up a stray!
On May 3, 2016 1:34 AM, "Сковорода Никита Андреевич" <
notifications@github.com> wrote:

> In tools/eslint/node_modules/graceful-fs/polyfills.js
> <https://github.com/nodejs/node/pull/6534#discussion_r61851725>:
>
> > @@ -1,5 +1,4 @@
> >  var fs = require('./fs.js')
> > -var constants = require('constants')
>
> I don't think this file should be changed in this commit.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6534/files/4c6c0c9b0898ef360b2ce648cf7cf41d3d22d560#r61851725>
>